lib/glific_web/resolvers/contacts.ex

```diff
@@ -55,14 +55,18 @@ defmodule GlificWeb.Resolvers.Contacts do
   @doc """
   Import contacts to the database
   """
-  @spec import_contacts(Absinthe.Resolution.t(), %{group_label: String.t(), data: String.t()}, %{
+  @spec import_contacts(Absinthe.Resolution.t(), map(), %{
           context: map()
         }) ::
           {:ok, any} | {:error, any}
-  def import_contacts(_, %{group_label: group_label, data: data}, %{
-        context: %{current_user: user}
-      }) do
-    Import.import_contacts(user.organization_id, group_label, data: data)
+  def import_contacts(_, %{id: id, type: type, group_label: group_label, data: data}, _) do
+    {:ok, org_id} = Glific.parse_maybe_integer(id)
+
+    case type do
```

Comment:
This looks reductant to me. 

can't we use guards in the Import.import_contacts function?

Reply:
@pankaj-ag made requested changes

Reply:
@pankaj-ag I also parse the id to integer, so using guard here then in every function I need to repeat same _parse_maybe_integer_.
Let me know your thoughts on same

Reply:
Glific.parse_maybe_integer(id)
|> elem(1)
|>  Import.import_contacts(group_label, [{type, data}])